package.json

```diff
@@ -9,8 +9,8 @@
     "watch": "export HOUDINI_WATCH=1; script/build.sh",
     "build": "script/build.sh",
     "build-all": "yarn --frozen-lockfile && script/compile-assets.sh && yarn build",
-    "build-all-staging": "export RAILS_ENV=staging; yarn --frozen-lockfile && script/compile-assets.sh && yarn build",
-    "build-all-production": "export RAILS_ENV=production; yarn --frozen-lockfile && script/compile-assets.sh && yarn build",
+    "build-all-staging": "export RAILS_ENV=staging; export NODE_ENV=development; yarn --frozen-lockfile && script/compile-assets.sh && yarn build",
+    "build-all-production": "export RAILS_ENV=production; export NODE_ENV=development; yarn --frozen-lockfile && script/compile-assets.sh && yarn build",
```

Comment:
The integration for yarn in assets:precompile needs a specific NODE_ENV to decide which packages to install for the build. Since we're running a build, we need need devDependencies installed, which means we need the NODE_ENV to be development